tony-core/src/main/java/com/linkedin/tony/ApplicationMaster.java

```diff
@@ -326,7 +326,11 @@ private boolean run(String[] args) throws IOException {
         .setStarted(started)
         .setUser(user);
     JobMetadata metadata = metadataBuilder.build();
-    eventHandler.setUpThread(jobDir, metadata);
+
+    if (!eventHandler.setUpThread(jobDir, metadata)) {
+      return false;
```

Comment:
shall we fail the job if this fails? i don't think this is fatal

Reply:
It returned false, when some exception occurred. I think it would be fine to avoid some unpredictable behavior latter (e.g., hanging for several hours without any allocation of resources)